src/rpc/misc.cpp

```diff
@@ -116,8 +116,9 @@ UniValue getinfo(const UniValue &params, bool fHelp)
     obj.push_back(Pair("status", statusStrings.GetPrintable()));
     obj.push_back(Pair("errors", GetWarnings("statusbar")));
     CBlockIndex *cashFork = chainActive.Tip();
+    const Consensus::Params &consensusParams = Params().GetConsensus();
     if (cashFork)
-        cashFork = cashFork->GetAncestor(BITCOIN_CASH_FORK_HEIGHT);
+        cashFork = cashFork->GetAncestor(consensusParams.uahfHeight);
     std::string fork = "Bitcoin";
```

Comment:
the code from line 121 to 130 writes "bitcoin" if you are pre-fork and "bitcoin cash" if you are post fork.  But at this point we are post fork -- I don't think that watching it switch is meaningful during IBD.  In fact its probably going to cause bugs because people will use this "fork" field to verify what client flavor is being used.

so we can delete 119 to 129 and replace 130 with: obj.push_back(Pair("fork", "Bitcoin Cash"));

Reply:
makes sense